### PR TITLE
WT-3110 New WT Utility tests

### PR DESCRIPTION
**Pull request by daveh86**
Not 100% ready yet.

I need to think about what else can be done with stats.

### CODE REVIEW

**Comment by daveh86** [comment]
@sulabhM can you take a look at this?

File: test/suite/test_util17.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+class test_util17(wttest.WiredTigerTestCase, suite_subprocess):
+    tablename = 'test_util17.a'
+
+    def test_stat_process(self):
+        """
+        Test alter in a 'wt' process
+        """
+        params = 'key_format=S,value_format=S'
+        outfile = "wt-stat.out"
+        expected_string = "cursor: cursor create calls=12"
```
Comment (sulabhM) [inline test/suite/test_util17.py]: Please put a comment why "12".

**Comment by daveh86** [comment]
@agorrod can you take a look at this?

File: test/suite/test_util17.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    def test_stat_process(self):
+        """
+        Test stat in a 'wt' process
+        This test is just here to confirm that stat produces a correct looking
+        output, it isn't here to do statistics validation.
+        """
+        params = 'key_format=S,value_format=S'
+        outfile = "wt-stat.out"
+        expected_string = "cursor: cursor create calls=12"
```
Comment (agorrod) [inline test/suite/test_util17.py]: This is the only thing that worries me in the test. Could you exclude the ``12``? I'm not sure why it is (or will always be) 12.

**Comment by agorrod** [comment]
Thanks @daveh86 lgtm

**Pull request merged**